.github/workflows/static_checks.yaml

```diff
@@ -43,40 +48,33 @@ jobs:
   clang-tidy-check:
     name: Clang-tidy Check
     runs-on: ubuntu-latest
+    container: greole/ofbase
     needs: [build-compilation-db]
     steps:
     - uses: actions/checkout@v4
-
-    - name: Add clang repo
-      run: |
-        sudo add-apt-repository 'deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-16 main'
-        wget https://apt.llvm.org/llvm-snapshot.gpg.key
-        sudo apt-key add llvm-snapshot.gpg.key
-
     - name: Install dependencies
       run: |
-        sudo apt update
-        sudo apt install \
-           clang-tidy-16 \
-           libomp-16-dev
-
+        apt update
+        apt install -y \
+           clang-tidy \
+           jq
     - uses: actions/download-artifact@v4
       with:
         name: build-artifact
         path: build
-
     - name: Run clang-tidy
       run: |
+        git config --global --add safe.directory /__w/OGL/OGL
         # Create list of .cpp files belonging to this repository
-        git ls-files | grep -E "\.(cpp)" > pattern
+        git ls-files | grep -E "\.(C)" > pattern
         # Create list of .cpp files that are in this repository and part of the
         # compilation database
         # also filters out " at the begin and end of the filename
-        jq ".[] | .file" build/ReleaseAll/compile_commands.json \
+        jq ".[] | .file" build/ReleaseCPUOnly/compile_commands.json \
```

Comment:
Does `ReleaseAll` mean it is for both CPU and GPU?
Why changes to `ReleaseCPUOnly`?

Reply:
No, the `ReleaseAll` came from the corresponding NeoFOAM workflow and just means build all targets (unit tests, core, ...)